models/collection.go

```diff
@@ -215,128 +201,15 @@ func (c *CollectionApi) body() (body []byte, err error) {
 	return
 }
 
-func (c *CollectionApi) shouldRetry(method string, statusCode int, err error) bool {
-	if handle_code, ok := c.CustomBehaviourCodes[statusCode]; ok && (handle_code == util.ReturnError || handle_code == util.Ignore) {
-		return false
-	}
-	return true
-}
-
-func (c *CollectionApi) requestWithRetry(ctx context.Context, method string, url string, body []byte) (statusCode int, responseBody []byte, err error) {
```

Comment:
Collection vs Base request with Retry Diff
https://www.diffchecker.com/TmzDy1lV/

Basically -- single difference is additional logic in base to override statusCode/response body with values returned from handleRequestError function (line 16-19).